xtable-core/pom.xml

```diff
@@ -61,6 +61,12 @@
             <groupId>org.apache.avro</groupId>
             <artifactId>avro</artifactId>
         </dependency>
+        <!-- https://mvnrepository.com/artifact/org.apache.parquet/parquet-avro -->
+        <dependency>
+            <groupId>org.apache.parquet</groupId>
+            <artifactId>parquet-avro</artifactId>
+            <version>1.15.0</version>
```

Comment:
Should we update the version defined in the parent pom? https://github.com/apache/incubator-xtable/blob/main/pom.xml#L152

We define the version there to keep it consistent between modules.